### DISCUSSION TITLE
GitHub Actions Workflow with DITA-OT - build fails when installing org.doctales.xmltask

### DISCUSSION THREAD

**Discussion by poraghal** [dita-ot/dita-ot#4220] [Q&A] [answered]
Last week, my GitHub Actions workflow suddenly started failing when installing the doctales plugin, did something change or does anyone have any ideas on how to troubleshoot this?

DITA-OT version 4.0.2

```
dita install org.doctales.xmltask
Error: Downloaded plugin file checksum 97b632928b74c37c6b55a45d9d26a636ffb355371286a66de54923fea134f866 does not match expected value 37c6508eeaabc09b87a44a9e50d6ac854da21b2df41508dfc21ed4db7b756469
```

**Comment by infotexture** [comment, marked as answer]
@poraghal Looks like the plug-in author @stefan-jung has moved the plug-in from its original location in the doctales organization to a new location.

See https://github.com/dita-ot/registry/issues/135#issuecomment-1614247729 & https://github.com/dita-ot/registry/pull/137.

**Reply by poraghal** [reply]
Thanks Roger, @infotexture updating the install command solved my problem, much appreciated.

I had seen that plugin, but thought it was a different one ;-).

**Reply by stefan-jung** [reply]
Sorry for the inconveniences caused, @poraghal and @infotexture .

**Reply by poraghal** [reply]
@stefan-jung Not at all, Stefan, thanks for your work on the software!
